docs/src/app_dev/daml_api/token_standard.rst

```diff
@@ -0,0 +1,276 @@
+a ``WildcardFilter`` can be included in the ``cumulative`` filter array:
+
+.. code-block:: json
+
+    {
+      "identifierFilter": {
+        "WildcardFilter": {
+          "value": {
+            "includeCreatedEventBlob": true
+          }
+        }
+      }
+    }
+
+The ``beginExclusive`` field is the offset from which to start reading transactions.
+To paginate, you can start with the ``participantPrunedUpToInclusive`` from ``GET ${PARTICIPANT_URL}/v2/state/latest-pruned-offsets``
+and continue by passing the offset of the last transaction from the previous response.
+
+Parsing the history
+^^^^^^^^^^^^^^^^^^^
+
+You can find an example parser `here <https://github.com/hyperledger-labs/splice/blob/main/token-standard/cli/src/txparse/parser.ts>`_.
+This handles transactions involving the ``Holding`` and ``TransferInstruction`` interfaces.
+
+The endpoint returns transaction trees as an array.
+The transactions are ordered as they occur in the ledger.
+Given an ``ExercisedEvent`` with ``nodeId=X`` and ``lastDescendantNodeId=Y``,
+the children of that node are those with ``nodeId`` in the range ``[X+1, Y]``.
+``CreatedEvent`` and ``ArchivedEvent`` (or equivalently, ``ExercisedEvent`` where ``consuming=true``) do not have children.
+
+Given the above, a tree-like traversal can be performed on the transaction nodes.
+Generally, a Token Standard parser will focus on the exercise of Token Standard choices and creation of contracts implementing Token Standard interfaces.
+Where further customization is required, a parser can decide to also focus on internal/specific choices that are not available in the standard, but in some specific implementation.
+
+In each Token Standard exercise node, one can find:
+
+* The choice being executed, useful to distinguish what operation was performed.
+* As part of the archival/creation of children, one can find out other relevant operations that happened. For example, creation or archival of ``Holdings``.
+* Meta key/values, of which part of the standard:
+
+  * ``splice.lfdecentralizedtrust.org/tx-kind``: the kind of operation happening in the node. This can give more information than the exercised choice does. It can be one of:
+
+    * ``transfer``
+    * ``merge-split``
+    * ``burn``
+    * ``mint``
+    * ``unlock``
+    * ``expire-dust``
+
+  * ``splice.lfdecentralizedtrust.org/sender``: which party is the sender in the node.
+  * ``splice.lfdecentralizedtrust.org/reason``: a text specifying the reason for the operation in the node.
+  * ``splice.lfdecentralizedtrust.org/burned``: how much of a holding was burned in the node.
+
+.. warning::
+
+    Meta key/values can be specified in several fields. For transfers, they should be merged in last-write-wins order of:
+
+    * event.choiceArgument?.transfer?.meta,
+    * event.choiceArgument?.extraArgs?.meta,
+    * event.choiceArgument?.meta,
+    * event.exerciseResult?.meta,
```

Comment:
what do the `?` marks mean? Probably that these fields are not guaranteed to be present?

I'd remove them and use the suggestion I gave above to clarify that they are optional.